sorc/ncep_post.fd/SURFCE.f

```diff
@@ -3826,7 +3831,7 @@ SUBROUTINE SURFCE
                jj = jsta+j-1
                do i=1,iend-ista+1
                ii = ista+i-1
-                 datapd(i,j,cfld) = GRID1(ii,jj)
+                 datapd(i,j,cfld) = IFI_APCP(i,jj)
```

Comment:
@SamuelTrahanNOAA should this be IFI_APCP(ii,jj)

Reply:
That's a disconcerting mistake. I thought I had updated all of the i-dimension decomposition changes, but it seems I missed one. Hopefully I didn't miss any others...

Reply:
I have corrected that in the branch, and I can confirm it will build with internal and external IFI on Hera.

Reply:
I just reread everything, and I don't see any indexing errors.